### PR TITLE
Changes in user api

### PR DESCRIPTION
**Pull request by SvetlanaTselikova**
https://github.com/malinink/easy-tests/issues/297

### CODE REVIEW

**Comment by SvetlanaTselikova** [comment]
@malinink Look,please

File: src/main/resources/api/api.yaml [hunk 1]
```diff
@@ -141,6 +204,26 @@ paths:
...
           schema:
             $ref: '#/definitions/V1Error'
+  /v1/users/me:
+    get:
+      summary: Info for current user
+      operationId: showCurrentUser
+      tags:
+        - users
+      responses:
+        "200":
```
Comment (malinink) [inline src/main/resources/api/api.yaml]: it's a good idea to use same quotes, I preferred double quotes

File: src/main/resources/api/api.yaml [hunk 2]
```diff
@@ -61,18 +69,80 @@ paths:
...
             $ref: '#/definitions/V1Error'
+        '403':
+          description: Permission denied
+          schema:
+            $ref: '#/definitions/V1Error'
         default:
           description: Unexpected error
           schema:
             $ref: '#/definitions/V1Error'
+  /v1/users/FindByState:
```
Comment (malinink) [inline src/main/resources/api/api.yaml]: both of methods should be in get, but query param should be not required.

**Review by malinink — CHANGES_REQUESTED** [review]

File: src/main/resources/api/api.yaml (hunk 1 above)
Comment (malinink) [reply src/main/resources/api/api.yaml]: Let me think a little, lets use single one.

**Comment by malinink** [comment]
@SvetlanaTselikova There are conflicts with current dev, merge and fix pls.

**Comment by malinink** [comment]
@SvetlanaTselikova seems to be OK

**Review by malinink — APPROVED** [review]

**Pull request merged**